crates/bevy_ecs/src/system/system_param.rs

```diff
@@ -5,6 +5,25 @@ use crate::{
 };
 use parking_lot::Mutex;
 use std::{any::TypeId, marker::PhantomData, sync::Arc};
+
+/// A parameter that can be used in a system function
+///
+/// # Derive
+/// This trait can be derived.
+///
+/// ```
+/// use bevy_ecs::prelude::*;
+/// use bevy_ecs::SystemParam;
```

Comment:
```suggestion
/// # use bevy_ecs::prelude::*;
/// # use bevy_ecs::SystemParam;
```
All other doc examples hide the use statements, so the same should be done here.

Reply:
I've hidden the prelude, but I kept the SystemParam one to show that it needs to be imported explicitly (i.e. it's not part of the prelude).